### DISCUSSION TITLE
There should be a possibility to add a short description and custom tag(s) to a cloud project

### DISCUSSION THREAD

**Discussion by kristofferwebmind** [umbraco/Umbraco.Cloud.Issues#822] [Ideas]
### Feature request
Requesting to add:

* A short description of a cloud project 
* A possibility to add one or more custom tags to a cloud project

### What? 
In the cloud portal, when an organization has many projects and many portal users, it is hard to get an overview over what's what. The only thing we have to go one is the name of the project. At the moment, we keep track of our projects' purpose and planned lifetimes in an excel spreadsheet. If there was a possibility to add some custom context to a project in the cloud portal, managing a bunch of projects would become much more of a joy. 

### Why?
It happens that we spin up tests projects that are sometimes later forgotten. It happens that we are invited to projects that are not owned by us. It even happens that an employee has created projects and that that employee is now no longer an employee... In short, we would not have to spend so much time reviewing what projects we actually have on cloud and what those projects are about if the projects had descriptions and tags.  

![image](https://github.com/umbraco/Umbraco.Cloud.Issues/assets/143802330/6847e8d2-0ca8-4d6a-9590-85c4a0f175a0)

**Comment by sajumb** [comment]
Hi @kristofferwebmind,
Thank you for reaching out with your feature request and the nice illustration.

Your suggestion to add a short description and the ability to assign custom tags to each cloud project is indeed insightful and aligns well with our mission to enhance and improve the project dashboard in the future. We have the intention to include filtering product versions, product types, project plans, and tags. While I can't give a specific timeline right now, we'll keep you updated on our progress.

**Comment by kristofferwebmind** [comment]
Hi, @sajumb! Thanks, that sounds great

**Comment by bjarnef** [comment]
I agree this would be very useful.
We have a few projects on Umbraco Cloud, which on daily basis is mentioned using a short name - some or them 3 letter abbreviation, but for technically reasons Umbraco Cloud requires at least project alias to be 4 characters.

As the number of projects on Umbraco Cloud within our organization grow, it would be useful to search/filter on these tags.

It could also be a tag like "headless", "education" or whatever makes sense for the specific organization.
